examples/sampling/adaptive-covariance-haario.ipynb

```diff
@@ -139,7 +139,7 @@
     "\n",
     "# Check convergence using rhat criterion\n",
     "print('R-hat:')\n",
-    "print(pints.rhat_all_params(chains))\n",
+    "print(pints.rhat(chains))\n",
```

Comment:
Thanks for changing these: what'd I'd prefer though would be to change all of these over to using the sampler results object (the one that prints a table of parameter quantiles). I think these are much neater and display everything that's needed. See the ``adaptive-covariance-haario-bardenet.ipynb`` for an example of this.